clients/egui/src/main.rs

```diff
@@ -21,17 +21,23 @@ fn main() {
         Err(err) => (Settings::default(), Some(err.to_string())),
     };
 
+    let icon_bytes = {
+        // experimentally-determined image correction
+        let mut this = include_bytes!("../lockbook.ico").to_vec();
+        this.chunks_exact_mut(4).for_each(|chunk| {
+            (chunk[0], chunk[1], chunk[2], chunk[3]) = (chunk[3], chunk[0], chunk[1], chunk[2]);
+        });
+        this.reverse();
+        this
+    };
+
     eframe::run_native(
         "Lockbook",
         eframe::NativeOptions {
             drag_and_drop_support: true,
             maximized: settings.window_maximize,
             initial_window_size: Some(egui::vec2(1300.0, 800.0)),
-            icon_data: Some(eframe::IconData {
-                rgba: include_bytes!("../lockbook.ico").to_vec(),
-                width: 32,
-                height: 32,
-            }),
+            icon_data: Some(eframe::IconData { rgba: icon_bytes, width: 128, height: 128 }),
```

Comment:
we had an 88x128 image or something and this said 32x32. I tried a 256x256 image and 256's here to match but it silently showed a placeholder icon. 128x128 works but showed up backwards and upside down and with wrong colors without the inexplicable and arbitrary transformation above.